packages/web/src/server/components/HTML/index.tsx

```diff
@@ -35,10 +35,10 @@ const HTML = (props: HTMLProperties) => {
 							content="width=device-width, initial-scale=1"
 						/>
 						<meta name="msapplication-TileColor" content="#2b2b2b" />
-						<meta
+						{/* <meta
```

Comment:
if we dont need this, can we remove this code?

Reply:
Yes will remove it after going through the usage of this file in detail.

Reply:
are we doing this?